src/StringTools/StringTools.csproj

```diff
@@ -14,6 +14,12 @@
 
     <AssemblyName>Microsoft.NET.StringTools</AssemblyName>
     <PackageDescription>This package contains the $(AssemblyName) assembly which implements common string-related functionality such as weak interning.</PackageDescription>
+    
+    <IncludeBuildOutput Condition="'$(TargetFramework)' == 'net35'">false</IncludeBuildOutput>
+    <!-- Don't publish the reference assembly if the build output isn't included. -->
+    <TargetsForTfmSpecificBuildOutput Condition="'$(IncludeBuildOutput)' == 'false'" />
+    <!-- NU5128: Add lib or ref assemblies for the net35 target framework. -->
+    <NoWarn>$(NoWarn),NU5128</NoWarn>
```

Comment:
I think we normally use semicolons. I imagine it's ok with mixed commas and semicolons, but it's good to be consistent.

Reply:
Thanks, updated to use a semicolon as the delimiter here. The official documentation states that you would use commas only for `vbproj` projects.